stacks/carpark-stack.js

```diff
@@ -79,41 +82,44 @@ export function CarparkStack({ stack }) {
     function: {
       environment: {},
       permissions: [indexerTopicQueue],
-      handler: 'event-bridge/eipfs-indexer.handler',
+      handler: 'event-bus/eipfs-indexer.handler',
     },
   }
 
-  const replicatorTarget = {
-    function: {
-      environment: {
-        SQS_REPLICATOR_QUEUE_URL: replicatorQueue.queueUrl,
+  /** @type {import('@serverless-stack/resources').EventBusQueueTargetProps} */
+  const targetReplicatorQueue = {
+    type: 'queue',
+    queue: replicatorQueue,
+    cdk: {
+      target: {
+        message: awsEvents.RuleTargetInput.fromObject({
+          region: awsEvents.EventField.fromPath('$.detail.region'),
+          bucket: awsEvents.EventField.fromPath('$.detail.bucketName'),
```

Comment:
Any reason not to match the original property name here? I am wary of renaming properties across events as it makes it harder to guess what a properties name will be when coding handlers for them.
```suggestion
          bucketName: awsEvents.EventField.fromPath('$.detail.bucketName'),
```